Cadence/index.ts

```diff
@@ -21,7 +21,7 @@ export namespace Cadence {
 	})
 	export const is = type.is
 	export const flaw = type.flaw
-	export function allocated(cadence: Cadence, date: isoly.Date, options?: { cap?: number }): number {
+	export function allocated(cadence: Cadence, date: isoly.Date, options?: { limit?: number }): number {
```

Comment:
I think we are going to need some comments explaining what the different parameters actually mean. For example, what date is it that we are giving the function?

Reply:
consider the same for all of the other functions that we have in this file that handles how sustainable something is

Reply:
date can not really mean many things here. it should be strait forward.